aws/resource_aws_default_route_table.go

```diff
@@ -160,8 +165,8 @@ func resourceAwsDefaultRouteTableCreate(d *schema.ResourceData, meta interface{}
 		return err
 	}
 
-	if v := d.Get("tags").(map[string]interface{}); len(v) > 0 {
-		if err := keyvaluetags.Ec2CreateTags(conn, d.Id(), v); err != nil {
+	if len(tags) > 0 {
+		if err := keyvaluetags.Ec2CreateTags(conn, d.Id(), tags); err != nil {
```

Comment:
i _think_ we'll need to pass in the map here `tags.Map()` instead of `tags` since the `Ec2CreateTags` method internally calls `New` on tags object

Reply:
Submitted https://github.com/hashicorp/terraform-provider-aws/pull/19011 to cover this nonintuitive situation 👍